### PR TITLE
Item card

### PR DESCRIPTION
**Pull request by Froggerson**
I added the item card component. Not sure how well it will work though. Be wary.

### CODE REVIEW

File: src/components/ItemCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          <div className ="rectangle bottom"></div>
+          <div className="image-container">
+            <img className="image" src="./cat-bread.jpg" alt="Lorem Ipsum"/>
+          </div>
+      </div>
+      <div className="info">
+        <p className="title">Cat Bread</p>
+        <div className="price">
+          <img className="diamond" src="Diamond.png" alt=""/>
+          <p clasName="amount">5,000</p>
```
Comment (annsudhart) [inline src/components/ItemCard/index.jsx:20]: line 20 - className, not clasName

File: src/components/ItemCard/style.less [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  .inside-rectangle{
+    display: block;
+    height:@item-rectangle-height;
+    width: @item-card-width;
+    display:flex;
+    text-align: center;
+
+    .rectangle{
+      position:absolute;
+      background: @gray-5;
```
Comment (annsudhart) [inline src/components/ItemCard/style.less:17]: `gray-5`? I don't see it defined anywhere

File: src/components/ItemCard/style.less [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      .image{
+        width: @item-image-width;
+      }
+    }
+  }
+  .info{
+    position: absolute;
+    top: 300px;
+    line-height: 28px;
+    letter-spacing: 0.03em
```
Comment (annsudhart) [inline src/components/ItemCard/style.less:51]: Forgot a semicolon at the end lol

File: src/components/ItemCard/index.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+          <div className ="rectangle top"></div>
+          <div className ="rectangle bottom"></div>
+          <div className="image-container">
+            <img className="image" src="./cat-bread.jpg" alt="Lorem Ipsum"/>
+          </div>
+      </div>
+      <div className="info">
+        <p className="title">Cat Bread</p>
+        <div className="price">
+          <img className="diamond" src="Diamond.png" alt=""/>
```
Comment (annsudhart) [inline src/components/ItemCard/index.jsx:19]: I suggest using an SVG instead of a PNG of the diamond (or maybe the 🔷emoji?)

**Comment by StormFireFox1** [comment]
This PR has been revised and moved to a new ticket, as this version of the PR has fallen behind upstream really badly.
You can find the new version at #355.

**Pull request closed without merging**